src/storage/controllers/storageController.ts

```diff
@@ -0,0 +1,23 @@
+import { Logger } from '@map-colonies/js-logger';
+import { RequestHandler } from 'express';
```

Comment:
run npm run lint:fix anf validate all files pass on git action:
1. tests (i think you use old node and its doesn't pass test on node 18
2. prittier + eslint -> npm run lint:fix